src/press/models.py

```diff
@@ -344,6 +346,21 @@ def navigation_items(self):
     def code(self):
         return 'press'
 
+    @property
+    def active_journals(self):
+        from journal import models as journal_models
+        # imported here to avoid circular imports
+        return journal_models.Journal.objects.filter(
+            hide_from_press=False,
+        )
```

Comment:
Django provides a model loader to avoid circular imports (and runtime imports):
`https://docs.djangoproject.com/en/3.2/_modules/django/apps/config/#AppConfig.get_model`

I also find the term `active` overlaps with its publication status (currently publishing or archived), perhaps `public_journals` or `listed_journals` would be a better name.

Reply:
Good suggestion--change made